school/views.py

```diff
@@ -1,3 +1,58 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect
+from django.views.generic import CreateView
+
+from school.forms import CreateSchoolForm
+from school.models import School
+
+
+def school_list(request):
+    return render(request, 'school_list.html', {'list': School.objects.all()})
+
+
+class CreateSchool(CreateView):
```

Comment:
CreateSchool class repeats the functionality of the school_create function. Please leave either class or function